src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -83,6 +83,11 @@ void BaseQNAcceleration::initialize(
                    pair.second->values().size(), pair.second->previousIteration().size());
   }
 
+  if (std::any_of(cplData.cbegin(), cplData.cend(), [](const auto &p) { return p.second->hasGradient(); })) {
+    PRECICE_WARN("Gradient data required by the configured mapping is not yet accelerated with quasi-Newton acceleration schemes, which might lead to numerical issues. "
```

Comment:
```suggestion
    PRECICE_WARN("Gradient data, which is required by at least one of the configured data mappings, is not yet compatible with quasi-Newton acceleration. This combination might lead to numerical issues. "
```

Reply:
To be reviewed https://github.com/precice/precice/pull/1373